examples/bri-3/src/bri/zeroKnowledgeProof/api/dtos/request/createAnchorHash.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { IsNotEmpty } from 'class-validator';
+import { BpiSubjectAccount } from '../../../../identity/bpiSubjectAccounts/models/bpiSubjectAccount';
+import { BpiAccount } from '../../../../identity/bpiAccounts/models/bpiAccount';
+import { DocumentObject } from '../../../models/document';
+
+// TODO: Revisit validations rules once we move into business logic implementation
+export class CreateAnchorHashDto {
+  @IsNotEmpty()
+  ownerAccount: BpiSubjectAccount;
+
+  @IsNotEmpty()
+  agreementState: BpiAccount;
+
+  @IsNotEmpty()
+  document: DocumentObject;
+
+  @IsNotEmpty()
+  signature: string;
```

Comment:
are we checking somewhere for the signature regex?

Reply:
Not yet. We can add validations in the upcoming PRs